packages/ui/src/composables/useEmitProxy.ts

```diff
@@ -1,21 +1,33 @@
-export const useEmitProxy = <T extends string>(events: T[]) => {
-  const createEmits = () => {
-    return events
-  }
+type Events = string | [string, string]
+
+const getEvent = (event: Events) => Array.isArray(event) ? event[0] : event
+const getEmit = (event: Events) => Array.isArray(event) ? event[1] : event
+
+export const useEmitProxy = (events: Events[]) => {
+  const createEmits = () => events.map(event => getEmit(event))
```

Comment:
`  const createEmits = () => events.map(getEmit)`

Reply:
Fixed it